src/linter/manifestJson/ManifestLinter.ts

```diff
@@ -119,4 +165,15 @@ export default class ManifestLinter {
 			}
 		});
 	}
+
+	detectDeprecatedPropertyNames(viewObject: Record<string, unknown>, pathToViewObject: string) {
+		// Detect every property name starting with "view" except "viewType":
+		for (const key of Object.keys(viewObject)) {
+			if (key.startsWith("view") && key !== "viewType") {
+				this.#reporter?.addMessage(MESSAGE.DEPRECATED_VIEW_CONFIG, {
```

Comment:
This does seem to produce some false-positive findings that cause issues when changing "viewName" to "name" in routing "targets". The application won't come up at all without an error unless sap.ui5/routing/config does specify `"type": "View"`.

We should improve our linting check and ideally the routing should log an error to explain why nothing is rendered.

cc @stopcoder

Reply:
The `type` option is mandatory when defining `Targets` in the routing config. This option is introduced with the new option set in routing. I think we need to enhance the hint by adding the information when options with prefix "view" are found saying that the "type": "View" needs to be added as well.

Reply:
Thanks for the insights. But what about the case when options are not prefixed? Isn't the "type" then also mandatory? With the view prefix it works without the "type".

Reply:
No. `type` is only needed for the new option set (the none-prefixed ones). `type` can also be provided in parallel to the prefixed options but it doesn't have any effect and routing doesn't complain about it.

Reply:
Yes, that's what I was saying. If the new option set is used without `type`, then nothing happens and the app does not render.

Reply:
See https://github.com/SAP/ui5-linter/pull/356